content/docs/guides/price-schedules.mdx

```diff
@@ -0,0 +1,132 @@
+
+> As a seller company, I sell two different memberships to my b2b ecommerce site.  The first is **Startup** and the second is **Enterprise**.  The **Enterprise** membership is considerably more expensive than the **Startup** membership, therefore I want to provide a slightly lower product pricing to my **Enterprise** customers.  Specifically, Jane Doe from Best Buy has signed up for an **Enterprise** membership, and John Deer from TechWorld has signed up for a **Startup** membership.
+
+How would we go about achieving this effect you might ask?  After reviewing the considerations, let's dive in!
+
+## Considerations
+
+1. In our example, we will be assigning price schedules to products and parties at the Buyer level, however you can get even more granular than that, and assign Product/Price Schedule combinations at the User Group level within a single Buyer.  This allows you further flexibility within your product visibility.
+
+2. This guide assumes that all products, categories and catalogs have been created successfully, with proper assignments.  Make sure that your product is assigned to a catalog that the buyer's users have access to, or you will have issues when trying to create your product assignments.
+
+
+## Create a Price Schedule
+
+First, lets create the price schedule that is to be associated with the Enterprise membership.  For the length of this example, we will use "USB Cord" as our Product.  
+
+Lets assume there is no tax or shipping costs, and the minimum order quantity is 1.  Our call to the OrderCloud API should like this (If successful, our call should return a `201 Created` response):
+
+```
+Method
+POST
+
+URL
+https://api.OrderCloud/v1/priceschedules
+
+Data
+{
+  "ID": "enterprise-priceschedule-id"
+  "Name": "USB Cord",
+  "MinQuantity": 1,
+  "PriceBreaks": [
+    {
+      "Quantity": 1,
+      "Price": 3.99
+    }
+  ]
+}
+
+Unflatten
+yes
+
+Headers
+application/json
+Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9...
+```
+
+Next, lets make the exact same call, but change the `PriceBreaks` field to adjust for the **Startup** pricing.  The `PriceBreaks` array in your second call should look like this: 
+
+```json
+{
+  ...
+  "PriceBreaks": [
+    {
+      "Quantity": 1,
+      "Price": 5.99
+    }
+  ]
+}
+```
+
+
+
+## Create a Product / Price Schedule Relationship
+
+Now that we have our two price schedules created, the next step is to create two different product assignments based on party.  By making these product assignments, we are creating a special relationships between the product, a price schedule and different parties.  Lets first connect the enterprise USB Cord price schedule and the Best Buy 'Buyer' party that is subject to **Enterprise** pricing.  Both of these `POST` calls should garner `204 No Content` responses if successful.
+
```

Comment:
I think it might be best to use `assign` instead of `connect`. If we start using many words for the same thing it can be confusing.

Reply:
Changed